raft/fsm.cc

```diff
@@ -44,10 +44,15 @@ fsm::fsm(server_id id, term_t current_term, server_id voted_for, log log,
         failure_detector& failure_detector, fsm_config config) :
         fsm(id, current_term, voted_for, std::move(log), index_t{0}, failure_detector, config) {}
 
-future<> fsm::wait_max_log_size(seastar::abort_source* as) {
+bool fsm::memory_limit_reached() const {
```

Comment:
What memory? The name should indicate that it is a log memory limit.

Reply:
aggree